src/app/zones/sources.ts

```diff
@@ -13,30 +14,19 @@ type DetailParams = {
   name: string
 }
 
-type EnvoyDataParams = DetailParams & {
-  dataPath: 'xds' | 'clusters' | 'stats'
-}
-
 type Closeable = { close: () => void }
 
 export type ZoneOverviewCollection = CollectionResponse<ZoneOverview>
 export type ZoneOverviewSource = DataSourceResponse<ZoneOverview>
 export type ZoneOverviewCollectionSource = DataSourceResponse<ZoneOverviewCollection>
 
-export type ZoneIngressSource = DataSourceResponse<ZoneIngress>
-export type ZoneIngressOverviewCollection = CollectionResponse<ZoneIngressOverview>
-export type ZoneIngressOverviewSource = DataSourceResponse<ZoneIngressOverview>
-export type ZoneIngressOverviewCollectionSource = DataSourceResponse<ZoneIngressOverviewCollection>
-
-export type ZoneEgressSource = DataSourceResponse<ZoneEgress>
-export type ZoneEgressOverviewCollection = CollectionResponse<ZoneEgressOverview>
-export type ZoneEgressOverviewSource = DataSourceResponse<ZoneEgressOverview>
-export type ZoneEgressOverviewCollectionSource = DataSourceResponse<ZoneEgressOverviewCollection>
-
 export type EnvoyDataSource = DataSourceResponse<object | string>
 
 export const sources = (api: KumaApi) => {
   return {
+    ...zoneIngresses(api),
+    ...zoneEgresses(api),
```

Comment:
We'll eventually do this by installing these as proper "micro-frontends" at a later date. These "destructures" just contain the red below on the left here, I just moved them into sub micro-frontends/modules.